src/notebook.envd

```diff
@@ -24,3 +24,32 @@ def jupyter_lab(envd_port=8888, host_port=0, token="''"):
             ]
         ]
     )
+
+def jupyter_notebook(envd_port=8888, host_port=8888, token="''"):
+    """Configure Jupyter Notebook.
+
+    Args:
+        envd_port (Optional[int]): port used by envd container (default=8888)
+        host_port (Optional[int]): port used by host, if not specified or equals to 0,
+            envd will randomly choose a free port, default is 8888
+        token (Optional[str]): access token
+    """
+    install.python_packages(["jinja2", "notebook"])
```

Comment:
I wonder why `jinja2` is required

Reply:
Just tried removing `jinja2`, there is no more errors.